gdpr_assist/templates/gdpr_assist/admin/action_anonymise.html

```diff
@@ -13,6 +13,12 @@
             {% endfor %}
         </ul>
 
+        The follow is a representation of the fields that will be cleared by anonymising:
```

Comment:
Not sure about that one, but?

```suggestion
        The following is a representation of the fields that will be cleared by anonymising:
```